.github/workflows/advanced.yml

```diff
@@ -97,7 +97,8 @@ env:
   SOURCE_PATH: ${{ github.workspace }}/code
   ARTIFACT_PATH: ${{ github.workspace }}/artifact
 
-  TEST_TOOLCHAIN: ${{ inputs.arch == 'aarch64' && inputs.platform == 'w64-mingw32' && inputs.crt == 'msvcrt' }}
+  TEST_TOOLCHAIN: ${{ (inputs.arch == 'aarch64' && inputs.platform == 'w64-mingw32' && inputs.crt == 'msvcrt') || (inputs.arch == 'x86_64' && inputs.platform == 'pc-cygwin') }}
```

Comment:
Why inputs.arch == 'x86_64'?

Reply:
This PR tests only `x86_64-pc-cygwin` target, as `aarch64-pc-cygwin` is not working at all and we don't have aarch64 runners with WSL.